iowrappers/redis_client.go

```diff
@@ -450,3 +451,18 @@ func (redisClient *RedisClient) PlanningSolutions(context context.Context, reque
 
 	return response, nil
 }
+
+// Directly reading redis with fixed key to get plan or place
+func (redisClient *RedisClient) FetchPlanPlace(context context.Context, redisKey string, response interface{}) error {
```

Comment:
Nit: Maybe rename to Fetch single Redis record?

Reply:
Done, changed to FetchSingleRecord()